// Copyright 2014 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Distilled article content.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package dom_distiller;

import "distilled_page.proto";

message DistilledArticleProto {
  // A title for the article.
  optional string title = 1;

  // The distilled pages in the article.
  repeated DistilledPageProto pages = 2;
}
